### DISCUSSION TITLE
OCR  of  condition areas,  and being  able  to use those  recognitions  for  logic

### DISCUSSION THREAD

**Discussion by invidtiv** [Nain57/Smart-AutoClicker#197] [General Feature Requests]
if by  some way you  could get the the program  to  read  a  score  and  then  use to compare to  other  scores in   screen  to  choose  by  condition greater than  lesser than  of  the scanned  values and  click accordingly to the condition.

**Comment by Nain57** [comment]
Closed as implemented in latest beta: https://github.com/Nain57/Smart-AutoClicker/releases/tag/4.0.0-beta01
